vimish-fold.el

```diff
@@ -415,6 +415,74 @@ This feature needs `avy' package."
            folds-before-point)))
       (message "No more folds before point"))))
 
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Folding marks
+
+(defcustom vimish-fold-marks '("{{{" . "}}}")
+  "Strings marking beginning and end of folding region.
+
+Automatically create folds from regions marked by `vimish-fold-marks' strings."
+  :tag "Fold marks."
+  :type '(cons string string)
+  :package-version '(vimish-fold . "0.2.3"))
+
+(defcustom vimish-fold-find-marks-on-open t
```

Comment:
What is the motivation for having this flag? I'd think that if the feature is enabled (already controlled by `vimish-fold-marks`), than it should scan on open, otherwise it should not.

Reply:
When `vimish-fold-find-marks-on-open` is set to `t` (default), marks will be added when file is opened. When it is set to `nli`, buffer will be scanned for marks only when `vimish-fold-from-marks` explicitly called by user. This flag was intended to disable introduced feature.

Reply:
Ah I see, in that case perhaps we don't need `vimish-fold-marks` to ever be `nil`. I was confused by one of your test where you set the marks to `'(nil . nil)`.

Reply:
>  perhaps we don't need `vimish-fold-marks` to ever be `nil`

I guess there's no harm in `vimish-fold-marks` validation. Just it case

Reply:
There is not much point in lavish checking in this sort of a language. You can never check everything, so just assume that what you are given is correct and fail if it is not :)